client/hooks/packs/useAddNewPack.js

```diff
@@ -1,6 +1,61 @@
+import { getQueryKey } from '@trpc/react-query';
+import { queryClient } from '~/constants/queryClient';
```

Comment:
if we need query client we should be importing the trpc one for consistency (not your fault i know its confusing how we have multiple created in app at the moment)

Reply:
I mistakenly added this i am only using use context majorly will remove it